src/sendResponse.js

```diff
@@ -0,0 +1,8 @@
+const sendResponse = (response, statusCode, data) => {
+  response.setHeader('Content-Type', 'application/json');
+  response.statusCode = statusCode;
+
```

Comment:
`You should respond with status 200 and OK status text.`
`If something is not correct, you should respond with 400 status, Bad request statusText`
You've forgotten about status text